Packages/com.unity.inputsystem/CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ however, it has to be formatted properly to pass verification tests.
 - Added `InputSystem.customBindingPathValidators` interface to allow showing warnings in the `InputAsset` Editor for specific InputBindings and draw custom UI in the properties panel.
 - Added `InputSystem.runInBackground` to be used internally by specific platforms packages. Allows telling the input system that a specific platform runs in background. It allows fixing of [case UUM-6744](https://issuetracker.unity3d.com/product/unity/issues/guid/UUM-6744).
 - Added new UIToolkit version of the `InputActionsAsset` editor. Currently this is incomplete (view-only) and the existing editor is still used by default.
+  - add functionality for actions, maps and bindings
```

Comment:
Not sure we need this granularity in the changelog as this isn't public unless someone alters a feature flag. I think we will just call out when its enabled with the edit support